File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rsp/asynchronousNotificationsofChanges/model/ServiceGuideChangeNotification.java
package org.ngbp.jsonrpc4jtestharness.rsp.asynchronousNotificationsofChanges.model;

import org.ngbp.jsonrpc4jtestharness.rsp.receiverQueryApi.model.ServiceGuideUrls;

import java.util.List;

public class ServiceGuideChangeNotification {
    public String msgType;
    public List<ServiceGuideUrls> urlList;
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/notification/model/ServiceGuideChangeNotification.kt
package org.ngbp.jsonrpc4jtestharness.rpc.notification.model

import org.ngbp.jsonrpc4jtestharness.rpc.notification.NotificationType
import org.ngbp.jsonrpc4jtestharness.rpc.receiverQueryApi.model.Urls

data class ServiceGuideChangeNotification (
        var urlList: List<Urls?>? = null
): RPCNotification(NotificationType.SERVICE_GUIDE_CHANGE)